src/main/resources/assets/paths/approveDataset.yaml

```diff
@@ -0,0 +1,49 @@
+put:
+  summary: DAC Chair updates dataset approval status
+  description: Endpoint allows DAC Chairs to submit approval decisions for datasets under their purview (or requested purview)
```

Comment:
I think it would be nice to add some commentary about the allowed operations here, i.e. a dataset's approval can go from null -> true/false, or false -> true, but not true -> false